src/WatchDogs.Contracts/IEntityFrameworkUnitOfWorkFactory.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace WatchDogs.Contracts;
+public interface IEntityFrameworkUnitOfWorkFactory
```

Comment:
The name binds the class to the EntityFramework but there is nothing related to it in the interface itself. How about renaming it to `IUnitOfWorkFactory`?

Reply:
Done.